src/components/RoundCount/RoundCount.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import styles from '../RoundCount/RoundCount.module.css';
+import { setRoundCount } from './RoundCountActions';
+
+export class RoundCount extends React.Component {
+    render() {
+        const options = [1, 2, 3, 4, 5, 6]
+        return (
+            <select input="How many rounds in your game?" className={styles.roundCountBox} data-cy="Drop-Down" onChange={(event) => this.props.setRoundCount(event.target.value)}>
```

Comment:
for the `data-cy="Drop-Down"`, can you get rid of the capitals, and also name it something a bit more descriptive to do with the round count?